preview-src/docs-roles.adoc

```diff
@@ -84,3 +84,22 @@ Weighted
 [.supported]
 Unweighted
 --
+
+== List of procedures
+
+.Neo4j procedures
+[options=header, cols="<70,<30"]
+|===
+| Name |
+
+| <<procedure_db_checkpoint, `db.checkpoint()`>>
+| +++
+<span class="label label--enterprise-edition">Enterprise Edition</span>
++++
+
+| <<procedure_db_clearquerycaches, `db.clearQueryCaches()`>>
+| +++
+<span class="label label--role-admin">@Admin</span>
```

Comment:
After discussion with @davidoliverSP2 , we decided "Admin only" would be a better title for the label.

```suggestion
<span class="label label--role-admin">@Admin</span>
```